de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/SarosComponent.java

```diff
@@ -47,7 +56,9 @@ public SarosComponent(final Project project) {
       LogLog.error("could not load saros property file 'saros.properties'", e);
```

Comment:
Check whether LogLog is required

Reply:
As it is currently not planned that this PR remains on the master, I would prefer not dealing with it here. It does not seem to cause problems.